### ISSUE TITLE
conda.yml - openssl=1.1.1=h7b6447c_0

### ISSUE THREAD

**Issue by 23saumyar** [aimagelab/show-control-and-tell#27]
Creating the environment with conda.yml was throwing errors at - openssl=1.1.1=h7b6447c_0. I changed it to - openssl=1.1.1g=h7b6447c_0 which worked.

**Comment by tejasri19** [comment]
> Creating the environment with conda.yml was throwing errors at - openssl=1.1.1=h7b6447c_0. I changed it to - openssl=1.1.1g=h7b6447c_0 which worked.

I got it run by replacing openssl package under pip in conda.yml

**Comment by amirunxiayang** [comment]
why loss = loss_cap + 4*loss_gate, I can not understand the 4
![image](https://user-images.githubusercontent.com/72070010/94533331-963be600-0271-11eb-9261-aa51c5fe9122.png)
